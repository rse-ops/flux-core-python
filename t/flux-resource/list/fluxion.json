{
  "all": {
    "version": 1,
    "execution": {
      "R_lite": [
        {
          "rank": "0",
          "node": "5b12c7ea7263",
          "children": {
            "core": "0-3"
          }
        },
        {
          "rank": "1",
          "node": "5b12c7ea7263",
          "children": {
            "core": "0-3"
          }
        },
        {
          "rank": "2",
          "node": "5b12c7ea7263",
          "children": {
            "core": "0-3"
          }
        },
        {
          "rank": "3",
          "node": "5b12c7ea7263",
          "children": {
            "core": "0-3"
          }
        }
      ],
      "starttime": 0,
      "expiration": 0
    }
  },
  "down": {
    "version": 1,
    "execution": {
      "R_lite": [
        {
          "rank": "3",
          "node": "5b12c7ea7263",
          "children": {
            "core": "0-3"
          }
        }
      ],
      "starttime": 0,
      "expiration": 0
    }
  },
  "allocated": {
    "version": 1,
    "execution": {
      "R_lite": [
        {
          "rank": "0",
          "node": "5b12c7ea7263",
          "children": {
            "core": "2-3"
          }
        },
        {
          "rank": "1",
          "node": "5b12c7ea7263",
          "children": {
            "core": "2-3"
          }
        }
      ],
      "starttime": 0,
      "expiration": 0
    }
  }
}
