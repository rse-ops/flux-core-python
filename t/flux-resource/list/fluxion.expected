     STATE PROPERTIES NNODES   NCORES    NGPUS
      free                 3        8        0
 allocated                 2        4        0
      down                 1        4        0
